<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>279</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change password</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="m_newPasswordTextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  color: #ef3131;
}</string>
     </property>
     <property name="text">
      <string>NEW PASSWORD</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="m_newPasswordConfirmationTextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  color: #ef3131;
}</string>
     </property>
     <property name="text">
      <string>CONFIRM</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="m_oldPasswordTextLabel">
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  color: #ef3131;
}</string>
     </property>
     <property name="text">
      <string>OLD PASSWORD</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLineEdit" name="m_newPasswordEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="m_oldPasswordEdit">
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLineEdit" name="m_newPasswordConfirmationEdit">
     <property name="styleSheet">
      <string notr="true">[errorState=&quot;true&quot;] {
  border-color: #ef3131;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="WalletGui::WalletCancelButton" name="m_cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="WalletGui::WalletOkButton" name="m_okButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::WalletOkButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletOkButton.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletCancelButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletCancelButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_oldPasswordEdit</tabstop>
  <tabstop>m_newPasswordEdit</tabstop>
  <tabstop>m_newPasswordConfirmationEdit</tabstop>
  <tabstop>m_okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_newPasswordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>checkPassword(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_newPasswordConfirmationEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>checkPassword(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_oldPasswordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>passwordChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkPassword(QString)</slot>
  <slot>passwordChanged(QString)</slot>
 </slots>
</ui>
